### DISCUSSION TITLE
Multi Window Support

### DISCUSSION THREAD

**Discussion by horaciodiez** [unoplatform/uno#8671] [Q&A] [answered]
Hi! Just a newbbie question. Is the new multi-windowing feature released in WindowsApp SDK 1.01 available to all Uno targetting OS?
I´ve seen this [#8341 ](https://github.com/unoplatform/uno/issues/8341) but i don´t know if it is the same request.
Thank you!

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question! It's not yet supported, but you can track the progress #8341 indeed.
